### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by malaykhakhar**

### CODE REVIEW

**Review by kaustubh-jsr — COMMENTED** [review]
This looks good to me.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FurnitureMart | Home</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="logo"><a href="#" class="remove-deco">FurnitureMart</a></h1>
```
Comment (git-0r) [inline index.html:12]: suggestion: i think here you are nesting too much.
and e-commerce page should also have categories, please check LMS to confirm.

**Review by git-0r — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (malaykhakhar) [reply index.html:12]: Done with the changes.

**Review by eakansh125 — APPROVED** [review]
I could not find any syntactic issues here.
Code looks clean, just make sure to provide value in alt of the images. Can be merged.

**Review by git-0r — COMMENTED** [review]
i see a lot of empty alt attributes, otherwise it's good.

**Pull request merged**